ocw/templates/ocw/instance_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 {% load render_table from django_tables2 %}
 {% load static %}
-{% load bootstrap4 %}
+{% load django_bootstrap5 %}
```

Comment:
houldn't it be `{% load django-bootstrap5 %}` ?

Reply:
> Shouldn't it be `{% load django-bootstrap5 %}` ?

No. See Shouldn't it be `{% load django-bootstrap5 %}` ?

Reply:
ok :)